### ISSUE TITLE
Validation set.

### ISSUE THREAD

**Issue by qkrdmsghk** [ratschlab/ncl#2]
I'm interested in your work and trying to run the code.
It seems that the MIMIC-III open-source dataset only provides a training set and a test set, no test set is provided.
How do I select the validation set? For example, the size of the validation set and the way to select it?

Thanks!

**Comment by hugoych** [comment]
Hi, 
As described in the [Data Gathering](https://github.com/ratschlab/ncl#data-gathering) section of the README, we use [MIMIC-III benchmark](https://github.com/YerevaNN/mimic3-benchmarks). It's a specific benchmark derived from the original data which provides a validation split. You can find all the pre-processing steps in the README

**Comment by tongziw** [comment]
Hi, as I checked the MIMIC-III benchmark repo, there wasn't a validation split produced by the procedures. Is it because they've updated the repo after your work? Also, the 'train_listfile.csv' is no longer available and now replaced by 'train/listfile.csv'.

**Comment by tongziw** [comment]
> 

I think I've figure out how to fix the problem above -- simply add `python -m mimic3models.split_train_val {dataset-directory}` would do the work.

**Comment by hugoych** [comment]
I just updated the README accordingly. Thank you !

**Comment by andrew-w-chen** [comment]
Hi everyone. I have a question about an issue I am encountering with the split_train_val file and I'm posting here in the hopes that one of you might know the answer:

When I run python -m mimic3models.split_train_val {dataset-directory} it creates an empty val_listfile.csv.

The problem seems to stem from an inconsistency in IDs under resources/valset.csv and /data/in-hospital-mortality/train/listfile.csv. Specifically, The IDs under resources/valset.csv:

100,1
1000,0
10003,0
10004,0
10006,0

Do not seem to match the IDs in /data/in-hospital-mortality/train/listfile.csv:

stay,period_length,stay_id,y_true
11269805_episode2_timeseries.csv,0,36738911,0
17543503_episode1_timeseries.csv,0,34151150,0
16225912_episode1_timeseries.csv,0,32470453,1
15437636_episode1_timeseries.csv,0,39956136,0

If you are able to provide any insight as to why this would be, I would be very appreciative.

Many thanks, and Happy New Year!
